go/grpcweb/wrapper_test.go

```diff
@@ -312,6 +311,14 @@ func (s *GrpcWebWrapperTestSuite) assertHeadersContainMetadata(headers http.Head
 	}
 }
 
+func (s *GrpcWebWrapperTestSuite) assertTrailersContainMetadata(trailers grpcweb.Trailer, meta metadata.MD) {
```

Comment:
I copied `assertTrailersContainMetadata` and `assertHeadersGrpcCode` from `assertHeadersContainMetaData` and `assertTrailersGrpcCode` because `http.Header.Get` maps passed key to MIME header.